contracts/test/EvilToken.sol

```diff
@@ -40,7 +42,8 @@ contract EvilToken is FaucetToken {
             destination: address(this),
             asset: address(this),
             amount: 1e6,
-            maxCalls: type(uint).max
+            maxCalls: type(uint).max,
+            collateralAsset: address(0)
```

Comment:
I think you can just use the existing `asset` parameter instead of introducing a new one.

Reply:
sure 👍